mob_skill_lists.sql

```diff
@@ -0,0 +1,3638 @@
+-- phpMyAdmin SQL Dump
```

Comment:
Okaaaaay
something went very wrong when you added that skill list, because github thinks you added 3,737 lines to that file and removed 8 other lines. What did you edit this file in, and how did you send said file to your fork?